101-batch-with-keyvault-pfx-password/README.md

```diff
@@ -0,0 +1,11 @@
+# ARM template for existing Azure Batch service with pfx from keyvault
+
+<a href="https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2Fazure%2Fazure-quickstart-templates%2Fmaster%2F101-batch-with-keyvault-pfx-password%2Fazuredeploy.json" target="_blank">
+    <img src="http://azuredeploy.net/deploybutton.png"/>
+</a>
+<a href="http://armviz.io/#/?load=https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-quickstart-templates%2Fmaster%2F101-batch-with-keyvault-pfx-password%2Fazuredeploy.json" target="_blank">
+    <img src="http://armviz.io/visualizebutton.png"/>
+</a>
+
+This template demostrates using Azure Batch service with pfx password certificate from keyvault
```

Comment:
is there a doc we can link to for more info?  ok if not, this is just pretty terse...

Reply:
Done. Nothing available as of now. So, changed statement for better understanding "This template demostrates use of pfx certificate from keyvault in Azure batch service."